www/docs/en/dev/cordova/events/events.md

```diff
@@ -323,6 +323,16 @@ function onBackKeyDown() {
 }
 ```
 
+### Windows Quirks
+
+Throw an error in a `backbutton` callback to force the default behavior:
+
+```javascript
+document.addEventListener('backbutton', function (evt) {
+    throw "Exit"; // This will suspend the app
```

Comment:
Why override the backbutton then? Should not overriding result in the default behavior anyway?